qgreenland/models/config/layer.py

```diff
@@ -57,6 +58,24 @@ class Layer(QgrBaseModel):
 
     _validate_description = reusable_validator("description", validate_paragraph_text)
 
+    @validator("style")
+    @classmethod
+    def style_file_does_not_contain_blocked_fonts(cls, value):
+        blocked_fonts = ["Cantarell", "Sans Serif"]
```

Comment:
Should `Helvetica` be added to this list? Are there only these few that cause problems?  Would it be more straightforward to have an allow-list? E.g., enforce that all fonts are Open Sans?

Reply:
Good question. I do think the allowlist is probably a better pattern.

`Helvetica` was actually never present in our style files. For some reason, QGIS was converting other font families to `Helvetica` when it applied the style. Very weird.

Reply:
b76dedcd